service11_function.py

```diff
@@ -0,0 +1,29 @@
+
+def getsubfunction(index):
+     if(index == 0):
+          return(0x01)
+     elif(index == 1):
+          return(0x02)
+     elif(index == 2):
+          return(0x03)
+     else:
+          return(0x00)
+     
+     
+        
+def form_reqmsg4srv11(session, sprmib_flag):    
+    sid = int("11", 17)

```

Comment:
Sir I realized that I had entered 17 directly as a decimal number instead of using base 16 for SID 11. I have now updated it to int("11", 16), which correctly converts to 17 in decimal. could you please confirm if this update is okay now?"